docs/site/development/development-process.md

```diff
@@ -96,7 +96,10 @@ When a ticket is in design, the owner is responsible for bringing back to the co
 
 Skipping test failures
 
-If there is a test failure that is due to a bug that cannot be fixed right now (eg it requires a data change, but it is past the data freeze), then use the logKnownIssue() method to disable the test. See http://site.icu-project.org/setup/eclipse/time for details, and look in the CLDR test code for some examples of usage.
+If there is a test failure that is due to a bug that cannot be fixed right now (eg it requires a data change, but it is past the data freeze), then use the logKnownIssue() method to disable the test. See <https://icu.unicode.org/setup/eclipse/time> for details, and look in the CLDR test code for some examples of usage.
```

Comment:
Question about the 'but it is past the data freeze'. I think we need to be more specific here, since we can make changes after the data freeze for issues during alpha & beta if the CLDR TC and the ICU TC agree.

Reply:
Maybe something like: 'when it is past the data freeze and the CLDR TC considers it too risk or not a priority to fix in the current release'.